src/chess/position.h

```diff
@@ -64,20 +64,12 @@ class Position {
 
   // Gets board from the point of view of player to move.
   const ChessBoard& GetBoard() const { return us_board_; }
-  // Gets board from the point of view of opponent.
-  const ChessBoard& GetThemBoard() const { return them_board_; }
-  // Gets board from the point of view of the white player.
-  const ChessBoard& GetWhiteBoard() const {
-    return us_board_.flipped() ? them_board_ : us_board_;
```

Comment:
(Out of scope of this PR, please ignore, just thoughts)

I believe flipped() should be called black_to_move().

Also, either it should be moved to Position, or even better, all fields from Position would be moved to Board, and Position would be removed.